### PR TITLE
[PAR-WDFT-911] Lucas Souleyrasse

### PR DESCRIPTION
**Pull request by LucasSlrs**
lab done but when I delete in my search bar it did not return to all products

### CODE REVIEW

**Comment by VProum** [comment]
Hi Lucas, 

Nice job overall, 
To solve the issue you have when you delete your search bar, you have to add some more information on your state of your filterableproduct component. 

you will have "products" which represent the whole list of products and "filteredproducts" which will represent the data that the other component will use as props and display in your views.

In that way, when you delete your search bar, you can go back to the "products" state instead and rerender the whole list!

Basically, you will never modify the "product" state, just the "filteredproduct" state

Vincent
